### PR TITLE
[DO-971] Updating DRV records

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: business_register/converter/pep.py [hunk 1]
```diff
@@ -253,6 +253,7 @@ def __init__(self):
             'pep_id',
             'business_register',
             'CompanyLinkWithPep')
+        self.invalid_data = 0
```
Comment (roman-tiukh) [inline business_register/converter/pep.py:256]: maybe? It is counter, not data.
```suggestion
        self.errors_count = 0
```

File: business_register/converter/pep.py [hunk 2]
```diff
@@ -708,8 +710,7 @@ def update(self):
         self.report.save()
 
         logger.info(f'{self.reg_name}: process() started ...')
-        self.report.errors = 0
-        PepConverterFromDB().process()
+        self.report.errors = PepConverterFromDB().process()
```
[the remark this answers is not in the document]
Comment (IegorSoboliev) [reply business_register/converter/pep.py]: fine)

File: business_register/converter/pep.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/converter/pep.py]: used counter in the naming

File: location_register/converter/drv.py [hunk 3]
```diff
@@ -202,7 +353,7 @@ def update(self):
         self.report.save()
 
         logger.info(f'{self.reg_name}: DrvConverter().process() started ...')
-        DrvConverter().process()
+        self.report.errors = DrvConverter().process()
```
[the remark this answers is not in the document]
Comment (IegorSoboliev) [reply location_register/converter/drv.py]: )

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**